core/envelope-sim/src/main/java/fr/sncf/osrd/envelope_sim/pipelines/MaxEffortEnvelope.java

```diff
@@ -45,6 +45,8 @@ public static Envelope addAccelerationAndConstantSpeedParts(
             );
             EnvelopeAcceleration.accelerate(context, 0, initialSpeed, overlayBuilder, 1);
             cursor.findPosition(overlayBuilder.getLastPos());
+            if (partBuilder.isEmpty())
+                throw new ImpossibleSimulationError(); // Not enough traction to start
```

Comment:
Is there a way we could send a specific error for this ? It's not a common running-time calculation error, it would be nice to know if the computation crashed because the rollingstock isn't powerful enough somewhere

Reply:
It's the purpose of this error, the full error message is `The requested train stopped before reaching its destination (not enough traction)`

Reply:
Aaah ok my bad